src/integration/pages/idxPage/canonicalTests.js

```diff
@@ -0,0 +1,47 @@
+import {
+  runCoreCanonicalTests,
+  runCanonicalAnalyticsTests,
+} from '../../common';
+
+export default () => {
+  runCoreCanonicalTests();
+  runCanonicalAnalyticsTests();
+
+  it('I can see a radio schedule component with an id', () => {
+    const hasRadioSchedule = service === 'persian';
+    const id = document.getElementById('Radio-Schedule');
+
+    if (hasRadioSchedule) {
+      expect(id).toBeInTheDocument();
+    } else {
+      expect(id).not.toBeInTheDocument();
+    }
+  });
+
+  describe('Story Promo', () => {
+    it('I can see an image', () => {
+      const section = document.querySelector('section');
+
+      if (section) {
+        expect(section).toBeInTheDocument();
+
+        const imageEl = section.getElementsByTagName('img')[0];
```

Comment:
As mentioned in my comment above, would this work for both canonical and amp (and therefore you could move this to crossPlatform):
        `const imageEl = section.getElementsByTagName('img amp-img')[0];`

Something similar was implemented for articles:
https://github.com/bbc/simorgh/blob/idx-integration-tests/src/integration/pages/articles/crossPlatformTests.js#L7

Reply:
Yeah I'll do that. I don't think we can select multiple tags with `getElementsByTagName` though, probably `querySelectorAll`.